<?xml version="1.0" encoding="UTF-8" ?>
<testsuite name="tst_Exception" timestamp="@TEST_START_TIME@" tests="2" failures="1" errors="0" time="@TEST_DURATION@">
  <properties>
    <property name="QTestVersion" value="@INSERT_QT_VERSION_HERE@"/>
    <property name="QtVersion" value="@INSERT_QT_VERSION_HERE@"/>
    <property name="QtBuild" value=""/>
  </properties>
  <testcase name="initTestCase" result="pass" time="@TEST_DURATION@"/>
  <testcase name="throwException" result="fail" time="@TEST_DURATION@">
    <failure result="fail" message="Caught unhandled exception"/>
  </testcase>
  <system-err/>
</testsuite>
